### PR TITLE
OCP4: Add rule to verify that the Compliance Operator is scanning

### PR DESCRIPTION
**Pull request by JAORMX**
This addresses RA-5 from the NIST SP 800 53 controls.

Note that this needs https://github.com/openshift/compliance-operator/pull/638
to be merged before working and a new release for the Compliance
Operator to persist that.

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

**Review by jhrozek — APPROVED** [review]
The code looks good, I'll let the merging up to you in case you wanted to cut the new release when the CO PR merges.

**Comment by JAORMX** [comment]
@jhrozek we have to wait for the CO PR to merge and a new release, since the e2e test will fail without this.

File: applications/openshift/risk-assessment/compliancesuite_exists/rule.yml
Comment (jhrozek) [inline applications/openshift/risk-assessment/compliancesuite_exists/rule.yml:21]: @JAORMX asked me to point out that CM-6 should be referenced here

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 7ae1fd58a7ad07f2e60a9447998c3a8624b0cbd9

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now b1f9810832f5c34c10c6830cbac5cc2eb6a377f7

File: applications/openshift/risk-assessment/compliancesuite_exists/rule.yml
Comment (jhrozek) [inline applications/openshift/risk-assessment/compliancesuite_exists/rule.yml:21]: Can you also add AC-17(2) ?

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now d38bc08d0b56d052a6d33a3c7b747754c14144ad

File: applications/openshift/risk-assessment/compliancesuite_exists/rule.yml
Comment (jhrozek) [inline applications/openshift/risk-assessment/compliancesuite_exists/rule.yml:21]: omg, I'm an idiot. I was looking at this PR and the FIPS one at the same time and for some reason thought this is the FIPS one which should reference AC-17(2) and already does actually. So this rule doesn't apply to AC-17(2) and the ref should be removed.
The rest looks good, feel free to just merge right away afterwards.

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 30f4587b28449a624ba5aa3fbd2ba5218b011691

**Review by jhrozek — APPROVED** [review]
Looks good!

**Pull request merged**
